htsohm/material_files.py

```diff
@@ -31,7 +31,7 @@ def random_number_density(number_density_limits, lattice_constants):
     min_atoms = int(min_ND * v)
     max_atoms = int(max_ND * v)
     if min_atoms < 2:
-        min_atoms = 2
+        min_atoms = int(2)
```

Comment:
A simpler way of doing this would just be: min_atoms = int(max(2, min_ND * v)).

or maybe: min_atoms = max(2, int(min_ND * v)).